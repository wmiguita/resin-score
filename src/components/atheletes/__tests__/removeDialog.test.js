import React from "react"
import renderer from "react-test-renderer"

import RemoveDialog from "../removeDialog"

describe( "RemoveDialog", () => {

  it( "should render RemoveDialog", () => {
    const athelete = { name: 'athelete to remove' }
    const tree = renderer.create( <RemoveDialog open={ false } athelete={ athelete } /> ).toJSON()

    expect( tree ).toMatchSnapshot()
  })
})
