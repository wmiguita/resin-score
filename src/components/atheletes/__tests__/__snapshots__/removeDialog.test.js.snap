// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RemoveDialog should render RemoveDialog 1`] = `null`;
